docs/services/data/_index.md

```diff
@@ -103,56 +105,57 @@ Export data with the Viam CLI and download your data for offline access.
 
 ### Upload a batch of data
 
-Upload data to the Viam app from your local computer or mobile device using the data client API, Viam CLI, or Viam mobile app.
+Upload data to the Viam app from your computer or mobile device using the data client API, Viam CLI, or Viam mobile app.
 
 {{% /manualcard %}}
 {{< /cards >}}
 
-#### Used with
+### Query your data
 
-You can configure the frequency of data capture individually for each supported component:
+Once your data has [synced](/services/data/capture-sync/), you can query it in multiple ways, including through the [data client API](/appendix/apis/data-client/) or [inside the Viam app](/how-tos/sensor-data-query-with-third-party-tools/).
+For _tabular_ sensor data, you can also [run {{< glossary_tooltip term_id="sql" text="SQL" >}} or {{< glossary_tooltip term_id="mql" text="MQL" >}} queries](/how-tos/sensor-data-query-sdk/) against your synced data from the [Query subtab](https://app.viam.com/data/query) of the **Data** tab in the Viam app.
 
 {{< cards >}}
-{{< relatedcard link="/components/arm/">}}
-{{< relatedcard link="/components/board/">}}
-{{< relatedcard link="/components/camera/">}}
-{{< relatedcard link="/components/encoder/">}}
-{{< relatedcard link="/components/gantry/">}}
-{{< relatedcard link="/components/motor/">}}
-{{< relatedcard link="/components/movement-sensor/">}}
-{{< relatedcard link="/components/sensor/">}}
-{{< relatedcard link="/components/servo/">}}
+{{% card link="/how-tos/sensor-data-query-with-third-party-tools/" %}}
+{{% card link="/how-tos/sensor-data-query-sdk/" %}}
+{{% card link="/appendix/apis/data-client/" %}}
 {{< /cards >}}
 
-## Query your data
-
-Once you have [synced](/services/data/cloud-sync/), you can query the data you've collected in multiple ways, including through the [data client API](/appendix/apis/data-client/) or [inside the Viam app](/how-tos/sensor-data-query-with-third-party-tools/).
-For _tabular_ sensor data, you can [run {{< glossary_tooltip term_id="sql" text="SQL" >}} or {{< glossary_tooltip term_id="mql" text="MQL" >}} queries](/how-tos/sensor-data-query-sdk/) against your synced data from the [Query subtab](https://app.viam.com/data/query) of the **Data** tab in the Viam app.
-
-## Permissions
+### Permissions
 
 Data management permissions vary between owners and operators.
 For more information about who can do what with data, see [Data Permissions](/cloud/rbac/#data-and-machine-learning).
 
 ## API
 
-The data management service supports the following methods:
+The [data client API](/appendix/apis/data-client/) supports the following methods:
+
+<br>
+
+Methods to upload data like images or sensor readings directly to the Viam app:
```

Comment:
flagging: not uploading data to the Viam App as defined as www.app.viam.com ; we shouldn't suggest this

Reply:
Better?
```suggestion
Methods to upload data like images or sensor readings directly to the Viam cloud:
```